ui/overview.js

```diff
@@ -66,10 +96,21 @@ function enable() {
     }
 
     addBackgroundMenu();
+
+    extensionStateChangedId =
+        Main.extensionManager.connect('extension-state-changed',
+            () => updateGhostPanelPosition());
+
+    updateGhostPanelPosition();
```

Comment:
I was thinking of doing this in the panel extension itself, but given you also have to move the panel ghost for the overview clone I guess this is a good place as well.